radix-engine/src/engine/call_frame.rs

```diff
@@ -104,4 +105,21 @@ impl CallFrame {
 
         Ok((taken, missing))
     }
+
+    pub fn auth_zone(&self) -> &AuthZone {
```

Comment:
Can we remove this from `CallFrame`? CallFrame should not there is such a thing as AuthZone. Better to have this function in AuthModule

Reply:
ahh, gotcha, sure